docker-compose.yml

```diff
@@ -56,24 +70,25 @@ services:
   repository-service-tuf-worker:
     build:
       context: .
-    entrypoint: "watchmedo auto-restart -d /opt/repository-service-tuf-worker -R -p '*.py' -- supervisord -c supervisor-dev.conf"
+    entrypoint: "bash entrypoint-dev.sh"
     environment:
       - DATA_DIR=./data
-      - RSTUF_STORAGE_BACKEND="LocalStorage"
-      - RSTUF_KEYVAULT_BACKEND="LocalKeyVault"
-      - RSTUF_LOCAL_STORAGE_BACKEND_PATH="/var/opt/repository-service-tuf/storage"
-      - RSTUF_LOCAL_KEYVAULT_PATH="/var/opt/repository-service-tuf/keystorage"
-      - RSTUF_BROKER_SERVER="amqp://guest:guest@rabbitmq:5672"
-      - RSTUF_REDIS_SERVER="redis://redis"
-      - RSTUF_WORKER_ID="dev"
+      - RSTUF_STORAGE_BACKEND=LocalStorage
+      - RSTUF_KEYVAULT_BACKEND=LocalKeyVault
+      - RSTUF_LOCAL_STORAGE_BACKEND_PATH=/var/opt/repository-service-tuf/storage
+      - RSTUF_LOCAL_KEYVAULT_PATH=/var/opt/repository-service-tuf/keystorage
+      - RSTUF_BROKER_SERVER=amqp://guest:guest@rabbitmq:5672
+      - RSTUF_SQL_SERVER=postgresql://postgres:secret@postgres:5432
```

Comment:
Shouldn't this be`5433`? Because of the port number update we do for the `postgres` service above?